<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>register_info</class>
 <widget class="QWidget" name="register_info">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>注册</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>员工ID：</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>员工姓名：</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLineEdit" name="IDlineEdit"/>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="n_lineEdit"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>员工部门：</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>电话号码：</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLineEdit" name="dp_lineEdit"/>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="ph_lineEdit"/>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>密码：</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>确认密码：</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLineEdit" name="pwd_lineEdit">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QLineEdit" name="agpwd_lineEdit"/>
   </item>
   <item row="6" column="0">
    <widget class="QPushButton" name="onButton">
     <property name="text">
      <string>注册</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QPushButton" name="offButton">
     <property name="text">
      <string>取消</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
